src/test/regress/citus_tests/join_gen/README.md

```diff
@@ -0,0 +1,221 @@
+## Query Generator for Postgres
```

Comment:
The folder structure of this new test suite is a little confusing to me. Some minor questions and suggestions:

- why is this under citus_tests? Can't we move one level up: `/src/test/regress/join_gen`
- Why do we call this as `join_gen`? Maybe make it more generic from now: `query_gen_test` or such?
- I'm fine with `join_gen/config` folder.
- However,  the existence of `join_gen/main.py` and test/citus_compare_dist_local_joins confusing to me. The latter is the main entry point for our cases, but normally one would expect the main.py to be the main entry point. So maybe:
           -  Rename main.py to something like: run_test_per_db.py 
           - Move run_test_per_db.py under "tests" folder
           - Rename "tests" folder to "bin" or such

Reply:
we reuse cluster creation, deletion etc.. defined in citus_tests.